### PR TITLE
Add missing steps to PassTicket configuration

### PR DESCRIPTION
**Pull request by janan07**
Describe your pull request here: This PR adds missing detail specific to each ESM

List the file(s) included in this PR: configuration-extender-passickets.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 1]
```diff
@@ -36,9 +36,42 @@ The following steps outline the procedure for enabling PassTicket Support:
...
-3. Enable the Zowe started task user ID to generate PassTickets for the API service. Grant `UPDATE` access to the Zowe started task by submitting commands in one of the three ESMs: ACF2, Top Secret, or RACF.
-4. Enable PassTicket support in the API Gateway for your API service.
+2. Activate the `PTKTDATA` class, which encompasses all profiles containing PassTicket information. Executing the following command:
+```
+SETROPTS CLASSACT(PTKTDATA) RACLIST(PTKTDATA)
+```
+
+3. Specify the application name requiring access through PassTicket for the UMS server with the following commands:
+```
+RDEFINE APPL UACC(NONE)
```
Comment (achmelo) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:46]: RDEFINE APPL `<application-name>` UACC(READ)

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 2]
```diff
@@ -36,9 +36,42 @@ The following steps outline the procedure for enabling PassTicket Support:
...
+SETROPTS CLASSACT(APPL)
+SETROPTS GENERIC(PTKTDATA)
+```
+
+Replace _NONE_ with a one to 8 character name designated for the application.
```
Comment (achmelo) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:51]: Replace `<application-name>` with ...

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 3]
```diff
@@ -36,9 +36,42 @@ The following steps outline the procedure for enabling PassTicket Support:
...
+Replace _NONE_ with a one to 8 character name designated for the application. 
+
+:::note
+This name is usually provided by the site security administrator.
+:::
+
+4. Define the profile for the application with the following command:
+
+```
+RDEFINE PTKTDATA SSIGNON(<key_description>())
```
Comment (achmelo) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:60]: RDEFINE PTKTDATA`<application-name>` SSIGNON(<key_description>())

**Review by achmelo — CHANGES_REQUESTED** [review]

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 4]
```diff
@@ -28,17 +28,69 @@ The API Gateway provides the user ID and password in the Authorization header of
...
 
-1. Follow the API service documentation that explains how to activate support for PassTickets.
+#### RACF
+
+1. Activate the `PTKTDATA` class, which encompasses all profiles containing PassTicket information. Executing the following command:
+```
+SETROPTS CLASSACT(PTKTDATA) RACLIST(PTKTDATA)
+```
+
+2. Specify the application ID requiring access through PassTicket for the UMS server with the following commands:
```
[the remark this answers is not in the document]
Comment (janan07) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md:38]: I added a link to the IBM doc for UMS

[the remark this answers is not in the document]
Comment (skurnevich) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md:66]: Isn't UMS server here is just an application example and should not be mentioned at all?

[the remark this answers is not in the document]
Comment (codezfire) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md:68]: I have updated UMS to ZOWE, used it as the reference to verify the correct configuration for a passticket setup.

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 5]
```diff
@@ -1,46 +1,108 @@
...
+
+```
+APPLDATA('NO REPLAY PROTECTION')
+```
+This links a secured sign-on application key with the application.
+:::
+
+4. Grant the Zowe user ID access to the application with the following command, and add the permitted user ID (In this case, the Zowe server user).
+```
+PERMIT APPLNAME CLASS(APPL) ID(<zowe-user-id>) ACCESS(READ)
```
Comment (cumarav) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:102]: should it be `<applid>`?

**Review by cumarav — CHANGES_REQUESTED** [review]

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md (hunk 5 above)
Comment (balhar-jakub) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md:102]: @Shobhajayanna Can you validate?

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 6]
```diff
@@ -1,46 +1,108 @@
...
+ Specifies the secured sign-on hexadecimal application key
+
+Replace with the application name defined previously.
+
+:::caution Important
+
+PassTickets for the API service must have the replay protection switched off. 
+
+```
+APPLDATA('NO REPLAY PROTECTION')
```
Comment (cumarav) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:95]: This line is part of upper command - it does not make any sense to split it into separate unrunable snippet. What user should do with it? Copy for what?

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 7]
```diff
@@ -1,46 +1,108 @@
...
+
+Replace _applid_ with a one to 8 character name designated for the application. 
+
+:::note 
+* This name is usually provided by the site security administrator.
+:::
+
+3. Define the profile for the application with the following command:
+```
+RDEFINE PTKTDATA <applid> SSIGNON(KEYMASKED(<key-description>))
```
Comment (cumarav) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:83]: return `APPLDATA('NO REPLAY PROTECTION')` to here after `<applid>`

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 8]
```diff
@@ -96,7 +158,16 @@ PERMIT IRRPTAUTH.<applid>.* CL(PTKTDATA) ID(<zowesrv>) ACCESS(UPDATE)
 SETROPTS RACLIST(PTKTDATA) REFRESH
 ```
 
-# Adding custom HTTP Auth headers to store user ID and PassTicket
+### Validate if the PassTicket Application is created
+
+```
+RLIST APPL APPLNAME ALL
```
Comment (cumarav) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:164]: Same bug here - should it be <applid> instead of `APPLNAME`

Comment (codezfire) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md:164]: yes we can generalize it as "applid" for this whole document

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md [hunk 9]
```diff
@@ -1,46 +1,108 @@
...
 
-- [Enabling PassTicket support](#enabling-passticket-support)
-- [Security configuration that allows the Zowe API Gateway to generate PassTickets for an API service](#security-configuration-that-allows-the-zowe-api-gateway-to-generate-passtickets-for-an-api-service)
+## Configuring Zowe to use PassTickets
 
-    - [ACF2](#acf2)
-    - [Top Secret](#top-secret)
-    - [RACF](#racf)
+Configuring Zowe to use PassTickets involves two processes:
+* Enabling PassTicket support for your ESM
```
Comment (codezfire) [inline docs/user-guide/api-mediation/configuration-extender-passtickets.md:31]: it should be 

"Enabling PassTicket support for your zowe appl"

File: docs/user-guide/api-mediation/configuration-extender-passtickets.md (hunk 5 above)
Comment (codezfire) [reply docs/user-guide/api-mediation/configuration-extender-passtickets.md:102]: we can generalize it as "applid" for this whole document

**Review by codezfire — APPROVED** [review]

**Review by achmelo — APPROVED** [review]

**Pull request merged**